frontend/src/tasks/tasks.html

```diff
@@ -0,0 +1,63 @@
+<div class="p-4 space-y-6">
+    <h1 class="text-2xl font-bold text-gray-700">Task Overview</h1>
```

Comment:
Dire need of loading spinner

![image](https://github.com/user-attachments/assets/56a9cb90-964f-4edc-aee7-49324e3e3b7d)